### PR TITLE
Feature/operator documents new notifications admin

### PR DESCRIPTION
**Pull request by tsubik**
Send notification to operator's responsible admins about new pending document ready for review.

PT: https://www.pivotaltracker.com/story/show/187126374

### CODE REVIEW

File: app/resources/concerns/operator_documentable.rb [hunk 1]
```diff
@@ -48,6 +48,11 @@ def attachment
 
     def attachment=(attachment)
       @model.build_document_file(attachment: attachment)
+      # TODO: check if there is better workaround for this issue https://github.com/rails/rails/issues/49898
```
Comment (santostiago) [inline app/resources/concerns/operator_documentable.rb:51]: 😮

File: config/locales/fr.yml [hunk 2]
```diff
@@ -383,7 +383,7 @@ fr:
       annexes: 'Annexes'
       operator_document_id: "ID de document de l'opérateur"
       with_deleted: 'Avec supprimé'
-      reason_label: "Raison de mettre le document comme “non applicable”"
+      reason_label: "Le document a été mis comme non applicable avec l'explication suivante"
```
Comment (santostiago) [inline config/locales/fr.yml:386]: 😆

**Review by santostiago — APPROVED** [review]
👍

File: app/resources/concerns/operator_documentable.rb (hunk 1 above)
Comment (tsubik) [reply app/resources/concerns/operator_documentable.rb:51]: Yeah, using too many AR callbacks could be a real trap in Rails project. Many times callbacks didn't work as expected.

**Pull request merged**